### PR TITLE
added SaveFileAsyncShouldReturnGuidWhenFileCreated

### PR DESCRIPTION
**Pull request by golibjon0226**

### CODE REVIEW

File: UnitTests/FileTests/FileServiceTests.cs [hunk 1]
```diff
@@ -75,9 +75,35 @@ public async Task SaveAsyncShouldThrowWhenDataIsNull()
...
+        var file = new Cambridge.Test.File.File(
+            type: EFileType.Document,
+            filename: guid,
+            extension: EAllowedExtension.txt,
+            data: byteData);
+        
+        // when
+        var fileId = await fileService.SaveAsync(file, CancellationToken.None);
+        var existingFilename = $"{fileId}.{file.Extension}";
+        var result = await fileService.ExistsAsync(existingFilename);
```
Comment (wahidustoz) [inline UnitTests/FileTests/FileServiceTests.cs]: var fileExists = await ...;

File: UnitTests/FileTests/FileServiceTests.cs [hunk 2]
```diff
@@ -75,9 +75,35 @@ public async Task SaveAsyncShouldThrowWhenDataIsNull()
...
+        
+        //should 
+        Assert.True(Guid.TryParse(fileId.ToString(), out Guid guidResult));
```
Comment (wahidustoz) [inline UnitTests/FileTests/FileServiceTests.cs]: Assert.True(fileId is Guid);

**Review by wahidustoz — CHANGES_REQUESTED** [review]

File: UnitTests/FileTests/FileServiceTests.cs (hunk 1 above)
Comment (golibjon0226) [reply UnitTests/FileTests/FileServiceTests.cs]: result is converted to fileExists

File: UnitTests/FileTests/FileServiceTests.cs (hunk 2 above)
Comment (golibjon0226) [reply UnitTests/FileTests/FileServiceTests.cs]: fixed

**Pull request closed without merging**